src/main/java/dk/bec/polonez/reservationsystem/service/FeatureService.java

```diff
@@ -26,7 +28,7 @@ public FeatureService(FeatureRepository featureRepository) {
         return featureRepository.findAll();
     }
 
-    public Feature getById(long id) {
+    public Feature getFeatureById(long id) {
         Optional<Feature> optionalFeature = featureRepository.findById(id);
         return optionalFeature.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
```

Comment:
**advice** Similar comment as in previous pull-requests: ```FeatureService`` is pure business logic class so it should be HTTP-protocol-agnostic - it shouldn't be aware of HTTP. Instead of throwing HTTP-specific exception:

```
new ResponseStatusException(HttpStatus.NOT_FOUND)
```

you should define your custom exception and throw it:

```
new FeatureNotFoundException()
```

and then add custom exception mapper (that would map exception to HTTP response code) in ```controller``` package.